### DISCUSSION TITLE
Syntax Color? Tweak?

### DISCUSSION THREAD

**Discussion by AJBcoding** [jwillinghalpern/filemaker-vscode-bundle#16] [Q&A]
Is there a way to change the syntax coloring?
I think I may have accidentally screwed mine up because it doesn't seem to be working the same as the images shared.

Thanks for any help. Great extension!

**Comment by jwillinghalpern** [comment]
Thanks!

You know, I noticed recently that my syntax highlighting didn't seem to work anymore too. I'll have to dig around and see why (or if you happen to take a look, lemme know if you spot something).

Sorry bout that!
J

**Comment by jwillinghalpern** [comment]
Hey, are you by any chance using Code Insiders? I noticed it doesn't work for me with Insiders, but still works with the regular version of VSCode.

**Comment by jwillinghalpern** [comment]
Hey @AJBcoding a couple things:

1. at the bottom right of your editor, make sure to select FileMaker as the coding language to enable syntax highlighting
2. If you're not seeing the colors for function names, try running `Developer: Reinstall Extension...` from the command palette and reinstall the FM VSCode bundle extension. That fixed the colors in VS Code Insiders Edition for me.

Hope one of those works for you.
